### PR TITLE
[ONNX] Opt into ruff fmt

### PR DESCRIPTION
**Pull request by justinchuby**
Add ONNX directory to use ruff format.

### CODE REVIEW

File: tools/linter/adapters/pyfmt_linter.py [hunk 1]
```diff
@@ -62,8 +62,9 @@
                     "torch/d*/**",
                     # torch/[e-n]*/**
                     "torch/[e-n]*/**",
-                    # torch/[o-z]*/**
-                    "torch/[o-z]*/**",
+                    "torch/optim/**",
```
Comment (XuehaiPan) [inline tools/linter/adapters/pyfmt_linter.py:65]:
```suggestion
                    # torch/optim/**
                    "torch/optim/**",
```

Add a line above to allow `torch/[e-n]*` and `torch/optim` can be migrated separately.

Comment (justinchuby) [reply tools/linter/adapters/pyfmt_linter.py:70]: Done

**Comment by justinchuby** [comment]
@XuehaiPan PTAL. I think it is ready to merge. Thanks!

**Review by XuehaiPan — APPROVED** [review]

**Comment by justinchuby** [comment]
@Skylion007 could you help approve? Thanks!

**Review by Skylion007 — APPROVED** [review]

**Comment by albanD** [comment]
@pytorchbot revert -m "Breaks main branch lint" -c nosignal

See https://hud.pytorch.org/pytorch/pytorch/commit/0870398fa8c3e097640f31cb8a8e2e2d3e522d33 for example failure

**Pull request closed without merging**